src/blocks/watson.rs

```diff
@@ -90,7 +90,7 @@ pub async fn run(config: Config, mut api: CommonApi) -> Result<()> {
         let state = read_to_string(&state_path)
             .await
             .error("Failed to read state file")?;
-        let state = serde_json::from_str(&state).error("Unable to deserialize state")?;
+        let state = serde_json::from_str(&state).or(serde_json::from_str("{}")).error("Unable to deserialize state")?;
```

Comment:
Instead of parsing a static string, it's better to `.unwrap_or` with a default value